### ISSUE TITLE
Feature Request: Longs as BigInts

### ISSUE THREAD

**Issue by jeffpyke** [stephenh/ts-proto#64]
Feature request: In addition to the Long library, add ability to type longs as [BigInts](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt).

**Comment by eba-ts** [comment]
+1. Many years ago we have chosen to code UUID as 2x64 bit integers. The option`--ts_proto_opt=forceLong=string` made it work, it would be nice to have BigInt.
```
message UUID {
    required fixed64 least_sign_bits = 1 
    required fixed64 most_sign_bits = 2 
}
```

**Comment by azizghuloum** [comment]
https://github.com/stephenh/ts-proto/pull/742

**Comment by stephenh** [comment]
This was fixed by #742 ; thanks @azizghuloum !

Also filed https://github.com/stephenh/ts-proto/issues/744 as a follow up.
